docs/http-useragent.rst

```diff
@@ -19,19 +19,6 @@ you should change the User-Agent to your application name and
 Customizing the User-Agent
```

Comment:
Setting `feedparser.USER_AGENT` should still work, so have left that section, but the `agent` parameter is no longer present